scripts/package

```diff
@@ -5,59 +5,6 @@ source $(dirname $0)/version
 
 cd $(dirname $0)/..
 
-DIST=$(pwd)/dist/artifacts
```

Comment:
isn't this useful for people testing airgap installs locally?

Reply:
`dist/artifacts` contains compressed archives of the `rke` binary, and are not used (are not release assets on GitHub). What or who will be missing these artifacts?

Reply:
I see, i thought this included the build.